examples/platform/bouffalolab/common/plat/platform.cpp

```diff
@@ -178,6 +178,7 @@ void UnlockOpenThreadTask(void)
 }
 #endif
 
+#if CONFIG_APP_FABRIC_TABLE_DELEGATE == 0
```

Comment:
would `!` work?

Reply:
changed to `#if !CONFIG_APP_FABRIC_TABLE_DELEGATE`